clang/test/dpct/cmake_migation/case_012/expected.txt

```diff
@@ -0,0 +1,7 @@
+add_library(culib)
+add_executable(cuexe)
+target_link_libraries(culib PUBLIC -L/lib/intel64 -lmkl_sycl -lmkl_intel_ilp64 -lmkl_tbb_thread -lmkl_core)
```

Comment:
Pls make sure the  mkl related libraries are in /lib/intel64, as far as I konw, if oneAPI pakage is installed, the mkl related libraries are in the installation directory of oneAPI, not in /lib/intel64.